spec/spec.md

```diff
@@ -1597,6 +1597,84 @@ JSONPath                      | Description
 
 ## Appendix
 
+### Security Considerations
+JSONPath and JSON schemas are being used in this specification. Both allow the use of Regular Expressions, which have 
+numerous different (often incompatible) implementations. Some implementations allow the embedding of arbitrary code, 
+which MUST never be permitted, given the presentation definition and submission data should be treated as input 
+from untrusted sources. 
+Regular expressions can often also be crafted to be extremely expensive to compute, resulting in
+denial-of-service attacks.
+
+As such this specification recommends not using Regular Expressions at all in your implementation when using JSONPath 
+and JSON Schemas, at the potential expense of expressiveness and/or usability. If you really want to use them, make sure
+the code is properly guarded both in terms of execution time and from a security perspective, by for instance 
+executing the code dealing with the regular expressions in a sandbox. 
```

Comment:
why not prohibit using it at all? what expressiveness/usability benefits are we aware of?

Reply:
Good question. It becomes harder to create certain more broad definitions. But as I argued in the PE meetings, I don't think that it would be good to say anything like 

'Just give me X, Y or Z, I don't really care where it comes from or what other constraints are present'

Normally a RP exactly knows that types of claims and Credentials it is requesting. You still have things like enumerations and constants etc. So IMO we could ditch using regexes entirely.